airflow/providers/amazon/aws/transfers/dynamodb_to_s3.py

```diff
@@ -137,8 +145,10 @@ def __init__(
         self.dynamodb_scan_kwargs = dynamodb_scan_kwargs
         self.s3_bucket_name = s3_bucket_name
         self.s3_key_prefix = s3_key_prefix
+        self.point_in_time_export = point_in_time_export
         self.export_time = export_time
         self.export_format = export_format
+        self.export_table_to_point_in_time_kwargs = export_table_to_point_in_time_kwargs
```

Comment:
this needs to be 

```
self.export_table_to_point_in_time_kwargs = export_table_to_point_in_time_kwargs or {}
```

otherwise if None is passed in, Line 199 fails because None can't be unpacked.

Reply:
fixed in https://github.com/apache/airflow/pull/41517